res/values/strings.xml

```diff
@@ -30,10 +30,14 @@
     <string name="refresh">Refresh</string>
     <!-- Strings related to Storing a location -->
     <string name="dialog_store_location_name">Enter a name for your current location:</string>
+    <string name="dialog_enter_location">Enter a name and gps coordinates for your location:</string>
```

Comment:
best to use 'GPS' in capitals, indicating it is an acronym (and for consistency with other strings where 'GPS' is used)